### ISSUE TITLE
Import new sampling data: Allmend-Brachy 2016

### ISSUE THREAD

**Issue by KW529** [felt0134/drought-net-tickets#7]
I am currently working to upload the cover data and then will move on to upload the biomass data.

**Comment by KW529** [comment]
Left off at line 238: select distinct taxa from cover_temp_kw order by taxa; #Checking misspellings

**Comment by KW529** [comment]
Need to clean up species list for cover- they list Forb, Graminoid, Legume, and Shrub

**Comment by KW529** [comment]
Aallmend-Brachy listed the following items and notes in 2016 taxa table (also 2015 & 2017 taxa files):
**Dead biomass:** group (litter   including dead wood)
**Standing biomass:** group (sum of live graminoid, forb, legume, shrub, moss, lichen)
**Vascular plant biomass:** group (sum of live graminoid, forb, legume, shrub)
**ANPP:** group (herbaceous live tissue of graminoid, forb, legume, shrub)
**Graminoid:** group (sum of Poaceae, Carex)
**Forb:** group (herbaceous species except graminoid, legume)
**Legume:** group (species of Fabaceae familiy)
**Shrub:** group (dwarf woody species including Helianthemum nummularium, Teucrium chamaedry, Teucrium montanum)
**Moss:** group
**Lichen:** group

**Comment by KW529** [comment]
QUESTIONS:
3. ARE THE FOLLOWING ITEMS ISSUES?
Trifolium montanum, Ranunculus species, Agrostis species (listed as Agrostis sp. in taxa table) listed in cover, but missing from taxa table
Agrostis sp., Agrimonia eupatoria, Centaurea sp., Parnassia palustris, Teucrium montanum, and Viola sp. in the taxa table, but not the cover table

**Comment by KW529** [comment]
CAPTAIN'S LOG,

It's been 3 hours and 8 minutes since I last saw any sign of hope as my struggle with SQL continues.

I experiences some concerning issues with the Aallmend-Brachy 2016 cover table omitting species when I joined my cover table with my taxa table. To fix the issue, I turned over a new leaf (or table, as it were). So, I dropped my cover table and created a new one, which worked. I also dropped my taxa table and have now encountered an SQL beast more ferocious than Cerberus. I now cannot upload any data to my taxa_info_temp_kw table; instead, I receive this cold message: ERROR 1406: Data _too_ long for column 'note_taxa' at row 2. 

This strikes me as odd, because I was able to upload this file to my taxa_info_temp_kw table just yesterday without strife. I updated the VARCHAR # for the note_taxa column to 200, then 300, then 1000. While the VARCHAR changed, ERROR 1406 continued to thwart my efforts.

My diagnosis: SQL is a pathological liar.
